src/App.tsx

```diff
@@ -1,56 +1,91 @@
-import React from 'react';
-import './App.scss';
-import { peopleFromServer } from './data/people';
+/* eslint-disable @typescript-eslint/quotes */
+import React, { useCallback, useMemo, useState } from "react";
+import "./App.scss";
+import classNames from "classnames";
+import debounce from "lodash/debounce";
+import { peopleFromServer } from "./data/people";
+import { Person } from "./types/Person";
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [query, setQuery] = useState("");
+  const [appliedQuery, setAppliedQuery] = useState("");
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+  const [showMenu, setShowMenu] = useState(false);
+
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const addQuery = useCallback(debounce(setAppliedQuery, 500), []);
+
+  const filteredPeople = useMemo(
+    () => peopleFromServer.filter(
+      person => person.name.toLowerCase().includes(appliedQuery.toLowerCase()),
+    ), [appliedQuery],
+  );
+
+  const handleQueryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
you should also change `showMenu` here to reach this requirement:
![image](https://github.com/mate-academy/react_autocomplete/assets/86923089/590947ee-5606-4cdd-a375-bc8a8467b352)

Reply:
I seem to have taken this point into account and am changing it in the input. Isn't that how it should be?

![image](https://github.com/mate-academy/react_autocomplete/assets/17146141/aadd99b5-f483-4913-80cc-7b68e86e7a0e)